tutorials/fundamentals/installation.md

```diff
@@ -58,15 +75,15 @@ conda-forge channel. You can install it with:
 conda install -c conda-forge napari
 ```
 
-### install from the master branch on Github
+### Install from the master branch on Github
 
 To install the "next-release" version from github via pip, call
```

Comment:
```suggestion
To install the "bleeding edge" version from github via pip, call
```

Reply:
I don't think "next-release" is very clear (but "bleeding edge" is also not very clear, so if you have any better ideas we should do that)

Reply:
Development version? Unstable version?